.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Pull Request
+about: A simple pull request template.
+
+---
+
+Provide a simple description of what the PR achieves. 
+
+## Pull Request Details
+
+Please ensure you PR fulfills the following criteria:
+
+- [ ] Does your PR solve an issue created by a member of staff or event handler?
+- [ ] Have you joined the Discord community? 
+- [ ] Were your changes made in the Pipenv environment?
+- [ ] Is your commit message clear and references the issue?
```

Comment:
Not sure this is required. PRs can have multiple commits & if some are lousily worded then I'm sure we can get by using the body of the PR and the changes made.